app/packages/core/src/components/ColorModal/colorPalette/AttributeColorSetting.tsx

```diff
@@ -56,24 +56,29 @@ const ChromePickerWrapper = styled.div`
 
 type ColorPickerRowProps = {
   style?: React.CSSProperties;
+  useLabelColors?: boolean;
 };
 
 type Input = {
   value: string;
   color: string;
 };
 
-const AttributeColorSetting: React.FC<ColorPickerRowProps> = ({ style }) => {
+
+const AttributeColorSetting: React.FC<ColorPickerRowProps> = ({
+  style,
+  useLabelColors,
+}) => {
   const pickerRef = useRef<HTMLDivElement>(null);
   const activeField = useRecoilValue(fos.activeColorField);
   const { colorPool, fields } = useRecoilValue(fos.sessionColorScheme);
   const setColorScheme = fos.useSetSessionColorScheme();
   const setting = useMemo(
-    () => fields.find((s) => s.path == (activeField as Field).path),
+    () => fields.find((s) => s.path == activeField.field.path),
```

Comment:
nit: could save as a const and reuse `const activeFieldPath = activeField.field.path`